shared/js/template.js

```diff
@@ -113,25 +113,43 @@
    *                          - safe, a list of properties that contain
    *                          HTML that is known and are
    *                          "known" to ignore.
-   */
-  Template.prototype.interpolate = function(data, options) {
-    // This _should_ be rewritten to use Firefox's support for ES6
-    // default parameters:
-    // ... = function(data, options = { safe: [] }) {
-    //
-    options = options || {};
-    options.safe = options.safe || [];
+  */
+  Template.prototype.interpolate = function(data = {}, options = { safe: [] }) {
```

Comment:
Agree, but it gives developers a better insight on the `interpolate` method signature, plus it is inlined with previous code where an empty Array is created if not given.
BTW I can remove it if you prefer.